docs/content/installing.rst

```diff
@@ -26,59 +26,76 @@ You can download the package manager and use it to install the dependencies abov
 Dependencies
 ============
 
-- `numpy <http://www.numpy.org>`_ 1.8 (or greater)
-- `scipy <https://docs.scipy.org/doc/scipy/reference>`_ 0.13 (or greater)
-- `matplotlib <https://matplotlib.org>`_ 1.3 (or greater)
-- `cython <https://cython.org/>`_ 0.20 (or greater)
-- `properties[math] <http://propertiespy.readthedocs.io>`_
+`discretize`'s runtime requirements are:
+
+- `numpy <http://www.numpy.org>`_ 1.22.4 (or greater)
+- `scipy <https://docs.scipy.org/doc/scipy/reference>`_ 1.8 (or greater)
+
+Additional functionality is provided when the following optional packages
+are installed:
+
+- `matplotlib <https://matplotlib.org/>`_
+- `pyvista <https://pyvista.org/>`_
+- `vtk <https://vtk.org/>`_
+- `omf <https://omf.readthedocs.io/en/latest/>`_
 
 We also recommend installing:
+
 - `pymatsolver <https://pymatsolver.readthedocs.io/en/latest/>`_ 0.1.2 (or greater)
 
 Installing discretize
 =====================
 
-discretize is on conda-forge::
+`discretize` is available on conda-forge and using the conda (or mamba) package manager
+is our recommended way of installing `discretize`::
 
     conda install -c conda-forge discretize
 
-discretize is on pypi::
+`discretize` is also available on pypi::
 
     pip install discretize
 
-.. attention:: Windows users
+There are currently pre-built wheels for windows available on pypi, but other operating
+systems will require a build.
 
-    If the pip install fails, please try installing the most recent version of
-    Visual Studio Community from https://visualstudio.microsoft.com/vs/community/
+Installing from Source
+----------------------
+If you are not a developer then conda-forge is really the preferred way of installing.
 
-    Within the **Python development** options, ensure that the following are included
+As `discretize` contains several compiled extensions and is not a pure python pacakge,
+installing `discretize` from the source code requires a C/C++ compiler capable of
+using a C++ 17 standard.
 
-        - Cookiecutter template support
-        - Python web support
-        - Python 3 64-bit
+`discretize` uses a `pyproject.toml` file to define the build and install steps. As such
+there is no `setup.py` file to run. You must use `pip` to install `discretize`. As long as
+you have an available compiler you should be able to install `discretize` from the source as::
 
-    .. image:: ../images/visual-studio-community.png
-        :align: center
+    pip install .
 
+Editable Installs
+^^^^^^^^^^^^^^^^^
+If you are an active developer of `discretize`, and find yourself modifying the code often,
+you might want to install if from source, in an editable installation. `discretize` uses
+`meson-python` to build the external modules and install the package. As such, there are several
+steps to take, first you must have the `meson-python` requirements in your environment::
 
+    pip install meson-python meson ninja
```

Comment:
Just merging the installation of build dependencies in a single line, and adding instructions for conda.

```suggestion
`meson-python` to build the external modules and install the package. As such, there are a few
steps to take. First, make sure you have the runtime dependencies installed in your environment (see Dependencies listed above). Then you must install some packages needed to build ``discretize`` in your environment. You can do so with ``pip``::

    pip install meson-python meson ninja cython setuptools_scm

Or with ``conda`` (or ``mamba``)::

    conda -c conda-forge install meson-python meson ninja cython setuptools_scm
```